packages/ember-metal/lib/property_events.js

```diff
@@ -254,8 +255,8 @@ function changeProperties(callback, binding) {
   }
 }
 
-function notifyBeforeObservers(obj, keyName) {
-  if (obj.isDestroying) { return; }
+function notifyBeforeObservers(obj, keyName, m) {
```

Comment:
how about `meta` (instead of `m`) for consistency with `dependentKeysDidChange` and `dependentKeysWillChange`?